app/models/user.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable,
+         :confirmable
+
+  has_many :budgets, foreign_key: :author_id
+  has_many :groups, foreign_key: :author_id
+
+  validates :name, length: { maximum: 100 }, presence: { message: 'El campo Nombre es obligatorio' }
+  validates :second_name, length: { maximum: 100 },
+                          presence: { message: 'El campo second nombre es obligatorio' }
```

Comment:
- [ ] With these validations, it was extremely or impossible :thinking:  for me to register and use your app. The messages are not clear to the user. Kindly make your login page user-friendly. 
- [ ] Once more kindly check the Entity Relationship Diagram that was given to us: The user should have a name. The second name is not really important. It renders the registration process more difficult.

Reply:
Done!